checker/tests/nullness-extra/issue348/Makefile

```diff
@@ -1,12 +1,13 @@
-.PHONY: all
+.PHONY: all clean
 
 all:
-	rm -f Out.txt
 	mkdir -p bin
 	$(JAVAC) lib/*.java -d bin/
-	rm -f bin/lib/Anno.class
 	jar cvf lib.jar -C bin/ lib/
 	$(JAVAC) -processor org.checkerframework.checker.nullness.NullnessChecker -Anomsgtext Issue348.java -d bin/ -cp lib.jar > Out.txt 2>&1
 	diff -u Expected.txt Out.txt
-	rm Out.txt lib.jar bin/*.class bin/lib/*.class
+
+clean:
+	rm -f bin/lib/Anno.class
```

Comment:
Is this needed? All classes are deleted below.

Reply:
Thanks, updated!